packages/block-editor/src/components/warning/style.scss

```diff
@@ -3,6 +3,7 @@
 	display: flex;
 	flex-wrap: wrap;
 	font-family: $default-font;
+	font-size: $default-font-size;
```

Comment:
We need to specify a fixed `font-size` here as the padding property for this class is relative to the font size.